diff --git a/node_modules/secp256k1/lib/index.js b/node_modules/secp256k1/lib/index.js
index 8187e17..dd2380c 100644
--- a/node_modules/secp256k1/lib/index.js
+++ b/node_modules/secp256k1/lib/index.js
@@ -19,7 +19,7 @@ function assert (cond, msg) {
 }
 
 function isUint8Array (name, value, length) {
-  assert(value instanceof Uint8Array, `Expected ${name} to be an Uint8Array`)
+  assert(value instanceof Uint8Array || value instanceof Buffer, `Expected ${name} to be an Uint8Array or Buffer`)
 
   if (length !== undefined) {
     if (Array.isArray(length)) {
